bluesky/suspenders.py

```diff
@@ -148,6 +158,11 @@ def __set_event(self):
                 loop = self.RE._loop
 
                 def local():
+                    ts = ((datetime.now() + timedelta(seconds=sleep))
+                           .strftime('%Y %m %d %H:%M:%S'))
+                    print("Suspender {!r} reports a return to nominal "
+                          "conditions. Will sleep for {!r} seconds and then "
+                          "release suspension at {!r}".format(self, ts))
```

Comment:
There are 3 format targets here, but only 2 inputs.  This is hanging because `local` is silently raising on an epics thread and the call-later is not going it.